src/components/Create/components/pages/ReviewDeploy/components/RewardsReview/RewardsReview.tsx

```diff
@@ -81,6 +94,13 @@ export const RewardsReview = () => {
             </div>
           }
         />
+        <DescriptionCol
```

Comment:
We weren't showing onchain goverance in NFT settings, so quickly added. dont @ me